pkg/shell/base_index.js

```diff
@@ -39,6 +39,8 @@ function Frames(index, setupIdleResetTimers) {
     if (!language)
         language = navigator.language.toLowerCase(); // Default to Accept-Language header
 
+    const lang_dir = cockpit.manifests.shell.rtl_locales.indexOf(language) > -1 ? "rtl" : "ltr";
```

Comment:
If you want to keep it in the shell manifest, then this needs to get along with it being absent. So please use `.?`, and make the `.indexOf()` (or better, `.contains()`) not crash on `undefined`.